src/server/cea_608_encoder/byte_pair_generator.py

```diff
@@ -132,7 +134,11 @@ def consume_captions(caption_list: list) -> dict:
     caption_metadata = {}
     caption_metadata['caption_string'] = []
 
+
     for caption in caption_list:
+        if 'caption_id' not in caption or 'caption_string' not in caption:
```

Comment:
The check for caption id and caption string should already exists, this looks like a duplicate check

Reply:
This was something that was already here before I started working on PER-397, but I can remove this.